path_rotate_root_test.go

```diff
@@ -0,0 +1,202 @@
+package azureauth
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test performs a rotate root operation and
+// swaps the root credentials for the application
+func TestRotateRootSuccess(t *testing.T) {
+	b, s := getTestBackend(t)
+
+	subscriptionID, tenantID, clientID, clientSecret := getAzureEnvironmentSettings()
+	if subscriptionID == "" || tenantID == "" ||
+		clientID == "" || clientSecret == "" {
+		t.Skip("environment variables not set, skipping test in CI")
```

Comment:
Nit: Was a little confused by "in CI" here when I ran this locally. Suggest to remove this mention since it can be ran outside of CI context as well.

Reply:
Good point, fixed error messaging in latest commit